community-operators/awss3operator/awss3operator.1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,135 @@
+    repository: https://github.com/yard-turkey/aws-s3-operator
+    alm-examples: '[{"apiVersion":"objectbucket.io/v1alpha1","kind":"ObjectBucketClaim","metadata":{"name":"myobc","namespace":"app-namespace"},"spec":{"generateBucketName":"mybucket-","bucketName":"my-awesome-bucket","storageClassName":"mystorageclass"}},{"apiVersion":"objectbucket.io/v1alpha1","kind":"ObjectBucket","metadata":{"name":"myobc-app-namespace-my-awesome-bucket","namespace":""},"spec":{"Connection":{"additionalState":{"ARN":"","UserName":""},"endpoint":{"additionalConfig":null,"bucketHost":"theendpoint.com","bucketName":"my-awesome-bucket","bucketPort":443,"region":"myregion","ssl":true,"subRegion":""}},"claimRef":"47ac88b7-7d6e-11e9-b8cf-42010af00007","reclaimPolicy":"Delete","storageClassName":"my-storageclass"}}]'
+    certified: "false"
+    support: OCTO
+spec:
+  displayName: AWS S3 Operator
+  description: |
+    AWS S3 Operator will deploy the AWS S3 Provisioner - more to be added
+  keywords: ['awss3provisioner', 'aws', 'S3', 'provisioner', 'buckets']
+
+  maintainers:
+  - name: Red Hat
+    email: openshift-operators@redhat.com
+
+  provider:
+    name: Red Hat
+
+  links:
+  - name: AWS S3 Provisioner
+    url: https://github.com/yard-turkey/awss3operator
+  - name: Documentation
+    url: https://github.com/yard-turkey/awss3operator
+
+  labels:
+    alm-status-descriptors: awss3operator.1.0.0
+    alm-owner-awss3: awss3operator
+
+  selector:
+    matchLabels:
+      alm-owner-awss3: awss3operator
+
+  icon:
+  - base64data: PHN2ZyB3aWR0aD0iMjQ5MCIgaGVpZ2h0PSIyNTAwIiB2aWV3Qm94PSIwIDAgMjU2IDI1NyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCI+PHBhdGggZD0iTTEyOC4wMDEuNjY3QzU3LjMxMS42NjcgMCA1Ny45NzEgMCAxMjguNjY0YzAgNzAuNjkgNTcuMzExIDEyNy45OTggMTI4LjAwMSAxMjcuOTk4UzI1NiAxOTkuMzU0IDI1NiAxMjguNjY0QzI1NiA1Ny45NyAxOTguNjg5LjY2NyAxMjguMDAxLjY2N3ptMCAyMzkuNTZjLTIwLjExMiAwLTM2LjQxOS0xMy40MzUtMzYuNDE5LTMwLjAwNGg3Mi44MzhjMCAxNi41NjYtMTYuMzA2IDMwLjAwNC0zNi40MTkgMzAuMDA0em02MC4xNTMtMzkuOTRINjcuODQyVjE3OC40N2gxMjAuMzE0djIxLjgxNmgtLjAwMnptLS40MzItMzMuMDQ1SDY4LjE4NWMtLjM5OC0uNDU4LS44MDQtLjkxLTEuMTg4LTEuMzc1LTEyLjMxNS0xNC45NTQtMTUuMjE2LTIyLjc2LTE4LjAzMi0zMC43MTYtLjA0OC0uMjYyIDE0LjkzMyAzLjA2IDI1LjU1NiA1LjQ1IDAgMCA1LjQ2NiAxLjI2NSAxMy40NTggMi43MjItNy42NzMtOC45OTQtMTIuMjMtMjAuNDI4LTEyLjIzLTMyLjExNiAwLTI1LjY1OCAxOS42OC00OC4wNzkgMTIuNTgtNjYuMjAxIDYuOTEuNTYyIDE0LjMgMTQuNTgzIDE0LjggMzYuNTA1IDcuMzQ2LTEwLjE1MiAxMC40Mi0yOC42OSAxMC40Mi00MC4wNTYgMC0xMS43NjkgNy43NTUtMjUuNDQgMTUuNTEyLTI1LjkwNy02LjkxNSAxMS4zOTYgMS43OSAyMS4xNjUgOS41MyA0NS40IDIuOTAyIDkuMTAzIDIuNTMyIDI0LjQyMyA0Ljc3MiAzNC4xMzguNzQ0LTIwLjE3OCA0LjIxMy00OS42MiAxNy4wMTQtNTkuNzg0LTUuNjQ3IDEyLjguODM2IDI4LjgxOCA1LjI3IDM2LjUxOCA3LjE1NCAxMi40MjQgMTEuNDkgMjEuODM2IDExLjQ5IDM5LjYzOCAwIDExLjkzNi00LjQwNyAyMy4xNzMtMTEuODQgMzEuOTU4IDguNDUyLTEuNTg2IDE0LjI4OS0zLjAxNiAxNC4yODktMy4wMTZsMjcuNDUtNS4zNTVjLjAwMi0uMDAyLTMuOTg3IDE2LjQwMS0xOS4zMTQgMzIuMTk3eiIgZmlsbD0iI0RBNEUzMSIvPjwvc3ZnPg==
```

Comment:
Your Operator carries the Prometheus logo?

Reply:
no, meant to replace, is there a common Red Hat logo that we can use, like one of the other Red Hat operators?